src/StoragelessSession/Session/DataScope.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace StoragelessSession\Session;
+
+class DataScope
+{
+    /**
+     * @var string
+     */
+    private $value;
+
+    /**
+     * @var string
+     */
+    private $key;
```

Comment:
No need to store this in the `DataScope` - it can live in `Data` (we'll also rename this stuff after we're done with the main functionality - `Data` is simply a silly name :-P )